APE14.rst

```diff
@@ -412,40 +578,43 @@ agreed as much as possible between different packages to make sure that these
 can be useful (which would not be the case if each package created their own
 set of custom type names).
 
-High-level Astropy Object
-^^^^^^^^^^^^^^^^^^^^^^^^^
+High-level API
+^^^^^^^^^^^^^^
 
-Unlike the low-level API, the 'high-level' interface described here will be a
-single Astropy-developed class since it interfaces with various Astropy objects.
-This high-level API would provide the ability for example to get ``SkyCoord``,
+The high-level API will provide the ability for example to get ``SkyCoord``,
```

Comment:
Add just a bit more context here.  Maybe "The high-level API's primary purpose is to provide fully-featured python objects for the world coordinates.  For example, to get ...<text already there>.  It is distinct from the actual implementation (discussed below) so that other objects can implement it on their own.  In this sence the astropy-provided implementation can be thought of as a reference implementation. "

Reply:
Fixed